### DISCUSSION TITLE
Installing with Homebrew can't download and install pkg-config

### DISCUSSION THREAD

**Discussion by jsale** [FluidSynth/fluidsynth#1184] [Q&A] [answered]
I'm getting the following when issuing either 'brew install fluidsynth' or 'brew install fluid-synth':

(base) Mac:~ jdoe$ brew install fluidsynth
==> Installing dependencies for fluid-synth: pkg-config, xz, cmake, gettext, glib
==> Installing fluid-synth dependency: pkg-config
==> Downloading http://pkgconfig.freedesktop.org/releases/pkg-config-0.28.tar.gz
######################################################################## 100.0%
curl: (60) SSL certificate problem: certificate has expired
More details here: https://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
HTTPS-proxy has similar options --proxy-cacert and --proxy-insecure.
Trying a mirror...
==> Downloading http://fossies.org/linux/misc/pkg-config-0.28.tar.gz
######################################################################## 100.0%
curl: (22) The requested URL returned error: 410 Gone
Error: Failed to download resource "pkg-config"
Download failed: http://fossies.org/linux/misc/pkg-config-0.28.tar.gz

**Comment by derselbst** [comment, marked as answer]
This is not the right place for this issue. You should report it to homebrew and ask them to update the URL to the pkg-config tarball.
